README.md

```diff
@@ -4,58 +4,48 @@ GraphQL based data sync server for mobile, with backend integration capabilities
 
 [![JavaScript Style Guide](https://img.shields.io/badge/code_style-standard-brightgreen.svg)](https://standardjs.com)
 
-## Trying it out
+## Getting Started
 
-*Start PostgreSQL & seed example data*
+### Install Dependencies
 
 ```
-npm i
-npm run db:start
-npm run db:seed
+npm install
 ```
 
-*Start Server*
+### Start and Initialize the Database
 
-```
-npm run dev
-```
-
-Go to http://localhost:8000/graphiql for an interactive query brower.
-The graphql endpoint is at `/graphql`.
-The subscriptions websocket is at `/subscriptions`.
-
-*Inspecting Postgres*
+Use docker compose to start the database.
 
 ```
-npm run db:shell
+docker-compose up db
 ```
 
-*Stop Server*
+Initialize the database in another terminal.
 
 ```
-# Ctrl-C to exit npm cmd
-npm run db:stop
+npm run db:init
 ```
 
-## Running on Kubernetes
+`npm run db:init` sets up the necessary tables and seeds the database with data useful for local development. **It is a destructive action.** It drops and recreates the tables every time.
 
-TODO
 
-## Architecture
+### Start the Server
```

Comment:
Can you add a "clean up" section like how to stop and remove Postgres container?

Reply:
I am not talking about hitting Ctrl+C to stop the container. Talking about `docker-compose rm` to remove the stopped containers since when you do `docker-compose up`, existing stopped containers are started again (feature of docker-compose).

Reply:
Great idea!